services/movie.py

```diff
@@ -28,13 +31,14 @@ def create_movie(
     genres_ids: list = None,
```

Comment:
For consistency and clarity, consider specifying the type of elements in the list, i.e., `list[int]` instead of just `list`. This matches the type hints used in the `get_movies` function.